sqlglot/dialects/dialect.py

```diff
@@ -27,6 +27,8 @@
 from sqlglot.trie import new_trie
 from sqlglot.typing import EXPRESSION_METADATA
 
+from importlib.metadata import entry_points
```

Comment:
Seems like [we should be good](https://docs.python.org/3/library/importlib.metadata.html):

> Added in version 3.8. (Note: changed in version 3.10: importlib.metadata is no longer provisional.)